tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -206,5 +208,26 @@ private static void WriteRequestHeaders(WebRequest request, IDictionary<string,
             var proxyInstance = obj.DuckAs<ILambdaContext>();
             return proxyInstance?.ClientContext?.Custom;
         }
+
+        internal static void InjectTraceContext(object executionContext, string traceId, string parentId, string samplingPriority)
```

Comment:
Same suggestion as below, you can re-use the proxy created in the instrumentation so you don't unnecessarily create a new `proxyInstance`

```suggestion
        internal static void InjectTraceContext<TExecutionContext>(TExecutionContext executionContext, string traceId, string parentId, string samplingPriority)
            where TExecutionContext : IExecutionContext
```

Reply:
implemented